### PR TITLE
Match a couple of Parakarry battle funcs

### PR DESCRIPTION
**Pull request by ThreeMustyFears**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.10%) on `us`.

File: src/battle/partner/parakarry.c [hunk 1]
```diff
@@ -1,7 +1,336 @@
...
+    ActorState* state = &parakarry->state;
+    Actor* targetActor;
+    ActorPart* targetActorPart;
+    f32 x, y, z;
+    s32 screenX, screenY, screenZ;
+    f32 tempX, tempY, tempZ;
+    f32 clampedAngleDiff, tempAngle;
+    s32 hudID, i;
+
+    if(isInitialCall) {
```
Comment (ethteck) [inline src/battle/partner/parakarry.c:46]:
```suggestion
    if (isInitialCall) {
```

File: src/battle/partner/parakarry.c [hunk 2]
```diff
@@ -1,7 +1,336 @@
...
+        evt->functionTemp[0] = 0;
+    }
+
+    switch(evt->functionTemp[0]) {
+        case 0:
+            hudID = hud_element_create(HudScript_AimMarkerF);
+            D_8023BD78[0] = hudID;
+            hud_element_set_render_depth(hudID, 0xA);
+            hud_element_set_render_pos(hudID, -0x64, -0x64);
```
Comment (ethteck) [inline src/battle/partner/parakarry.c:55]: positions and depths in decimal please

File: src/battle/partner/parakarry.c [hunk 3]
```diff
@@ -1,7 +1,336 @@
...
+            state->currentPos.x = parakarry->currentPos.x + 8.0f;
+            state->currentPos.y = parakarry->currentPos.y + 16.0f;
+            state->currentPos.z = parakarry->currentPos.z;
+            state->angle = atan2(state->currentPos.x, state->currentPos.y, state->goalPos.x, state->goalPos.y);
+            state->bounceDivisor = state->angle;
+            state->distance = 116.0f;
+            state->unk_18.x = state->angle;
+            i = 0;
+
+            do {
```
Comment (ethteck) [inline src/battle/partner/parakarry.c:148]: Can this be a for loop with two exit conditions (i < 30, screenY < 181)? I feel like that'd be nicer (also applies below)

File: src/battle/partner/parakarry.c [hunk 4]
```diff
@@ -1,7 +1,336 @@
...
+                tempAngle = clamp_angle(state->unk_18.y);
+                x = state->currentPos.x;
+                y = state->currentPos.y;
+                z = state->currentPos.z;
+                add_vec2D_polar(&x, &y, state->distance, tempAngle);
+                get_screen_coords(gCurrentCameraID, x, y, z, &screenX, &screenY, &screenZ);
+                i++;
+            } while (screenY >= 30 && i < 30);
+
+            state->moveTime = 0x78;
```
Comment (ethteck) [inline src/battle/partner/parakarry.c:174]: non-flag values in decimal (also applies below with 0x5A, 0x3C

File: src/battle/partner/parakarry.c [hunk 5]
```diff
@@ -1,7 +1,336 @@
...
+                    break;
+                }
+            }
+
+            sfx_play_sound(SOUND_311);
+            D_8023BDBC = 0x3C;
+            evt->functionTemp[0] = 2;
+            break;
+        case 2:
+            if(!(gActionCommandStatus.autoSucceed)) {
```
Comment (ethteck) [inline src/battle/partner/parakarry.c:207]: space after if before open paren (applies above and below as well)

**Review by ethteck — CHANGES_REQUESTED** [review]
Nice! A few style things (numbers in decimal, space after if mostly)

File: src/battle/partner/parakarry.c [hunk 6]
```diff
@@ -1,7 +1,336 @@
...
+    s32 screenX, screenY, screenZ;
+    f32 tempX, tempY, tempZ;
+    f32 clampedAngleDiff, tempAngle;
+    s32 hudID, i;
+
+    if(isInitialCall) {
+        evt->functionTemp[0] = 0;
+    }
+
+    switch(evt->functionTemp[0]) {
```
Comment (nanaian) [inline src/battle/partner/parakarry.c:50]:
```suggestion
    switch (evt->functionTemp[0]) {
```

File: src/battle/partner/parakarry.c [hunk 7]
```diff
@@ -1,7 +1,336 @@
...
+        case 0:
+            hudID = hud_element_create(HudScript_AimMarkerF);
+            D_8023BD78[0] = hudID;
+            hud_element_set_render_depth(hudID, 0xA);
+            hud_element_set_render_pos(hudID, -0x64, -0x64);
+
+            hudID = hud_element_create(HudScript_AimMarkerE);
+            D_8023BD78[1] = hudID;
```
Comment (nanaian) [inline src/battle/partner/parakarry.c:58]: Might be possible to compress this to
```suggestion
            D_8023BD78[1] = hudID = hud_element_create(HudScript_AimMarkerE);
```

File: src/battle/partner/parakarry.c [hunk 8]
```diff
@@ -1,7 +1,336 @@
...
+            tempAngle = clamp_angle(state->angle);
+            tempX = state->currentPos.x;
+            tempY = state->currentPos.y;
+            tempZ = state->currentPos.z;
+            add_vec2D_polar(&tempX, &tempY, state->distance, tempAngle);
+            z = state->currentPos.z;
+            x = state->currentPos.x;
+            y = state->currentPos.y;
+
+            for(i = 0; i < 6; i++) {
```
Comment (nanaian) [inline src/battle/partner/parakarry.c:317]:
```suggestion
            for(i = 0; i < ARRAY_COUNT(D_8023BD98); i++) {
```

**Review by nanaian — CHANGES_REQUESTED** [review]
Nice pr!

func_80238E24_700BA4 also needs space after keywords if/switch

**Pull request merged**